GG/src/Edit.cpp

```diff
@@ -449,9 +451,11 @@ void Edit::KeyPress(Key key, boost::uint32_t key_code_point, Flags<ModKey> mod_k
         } else if (0 < m_cursor_pos.second) {
```

Comment:
some comments in here explaining what you're doing / is happening would be helpful...